sound/soc/sof/ipc4-topology.h

```diff
@@ -26,4 +28,79 @@ struct sof_ipc4_pipeline {
 	struct sof_ipc4_msg msg;
 };
 
+/**
+ * struct sof_ipc4_available_audio_format - Available audio formats
+ * @base_config: Available base config
+ * @out_audio_fmt: Available output audio format
+ * @ref_audio_fmt: Reference audio format to match runtime audio format
```

Comment:
missing kernel-doc fields for dma_buffer_size and audio_fmt_num. Can be done in fixup